src/components/NavTabs.js

```diff
@@ -17,64 +18,73 @@ const styles = theme => ({
   content: {
     flexGrow: 1,
     padding: theme.spacing.unit * 3
-  },
-  toolbar: theme.mixins.toolbar
+  }
+})
+
+const TabContainer = withStyles(styles)(props => {
+  const { children, classes, display } = props
+  return (
+    <main className={classes.content} style={{ display }}>
+      <Typography component="div">{children}</Typography>
+    </main>
+  )
 })
 
+TabContainer.propTypes = {
+  children: PropTypes.node.isRequired,
+  classes: PropTypes.object,
+  display: PropTypes.string
+}
+
 class NavTabs extends React.Component {
   static propTypes = {
     classes: PropTypes.object
   }
 
   state = {
-    value: 0
+    activeTab: 0
   }
 
-  handleChange = (event, value) => {
-    this.setState({ value })
+  handleTabChange = (event, activeTab) => {
+    this.setState({ activeTab })
   }
 
   render() {
     const { classes } = this.props
-    const { value } = this.state
+    const { activeTab } = this.state
 
     return (
       <div className={classes.root}>
         <CssBaseline />
         <AppBar position="fixed" className={classes.appBar}>
           <Tabs
-            value={value}
-            onChange={this.handleChange}
+            value={activeTab}
+            onChange={this.handleTabChange}
             textColor="primary"
             indicatorColor="primary"
           >
             <Tab label="Nodes" />
-            <Tab label="Network" />
-            <Tab disabled label="Transactions" />
-            <Tab label="Tools" />
+            <Tab label="Network" disabled />
+            <Tab label="Transactions" disabled />
+            <Tab label="Tools" disabled />
           </Tabs>
         </AppBar>
 
-        {value === 0 && <NodesTab />}
+        <div style={{ display: activeTab === 0 ? 'inherit' : 'none' }}>
+          <NodesTab />
+        </div>
+
+        <TabContainer display={activeTab === 1 ? 'block' : 'none'}>
```

Comment:
Not a fan of this pattern that affects `display` values and abstracts some of that away into TabContainer. What didn't you like about the conditional rendering? e.g. `{activeTab === 0 && <NodesTab />}`